using-timescaledb/telemetry.md

```diff
@@ -18,6 +18,7 @@ the JSON that is sent to our servers about a specific deployment:
 	"install_method": "source",
 	"os_name": "Darwin",
 	"os_version": "Darwin Kernel Version 17.5.0: Mon Mar  5 22:24:32 PST 2018",
+	"os_name_pretty":"Debian GNU/Linux 8 (jessie)",
```

Comment:
Hmm, the problem is this example is internally inconsistent (i.e. os_name is `Darwin` but `os_name_pretty` is for Debian). Might want to copy the other fields from the example I gave.

(Also put a space after the `:` for readability)

Reply:
Ok! Updated